packages/query-graphql/src/decorators/graphql-resolve-info.utils.ts

```
@@ -68,7 +68,18 @@ function parseFieldNodes<DTO>(
 ): QueryResolveTree<DTO> | QueryResolveFields<DTO> {
   const asts: ReadonlyArray<SelectionNode> = Array.isArray(inASTs) ? inASTs : [inASTs]
 
-  return asts.reduce((tree, fieldNode) => {
+  const astsWithInlinedFragments = asts.flatMap(ast => {
```

ESLint

[error] Replace `ast` with `(ast)`

(prettier/prettier)